### ISSUE TITLE
[Feature]: Config option / MK2 unit / enchant to make space breathing unit / enchant not need Oxygen Can

### ISSUE THREAD

**Issue by MissCatTrap** [gisellevonbingen-Minecraft/Ad-Astra-Giselle-Addon#62]
### Cause for request

At the moment, if you do not use Ad Astra late game armour you are forced to forever carry Oxygen Cans. While they can go in your curios slot, their limited capacity makes them inconvenient, especially in late game mod runs (such as Mekanism) where you unlock the ability to create fusion reactors and creative flight, but have no late game alternative to Oxygen Cans.

### Request details

Enable a config option to make Space Breathing Unit / Enchant not require an enabled Oxygen can in players inventory (unlimited oxygen). This is an add-on for the difficult, expensive, late game MekaSuit (and the enchant can be made more expensive / a MK2 version e.g. after Killing Wither) that will make players lives more convenient. it will still encourage base oxygen supplies for friends, farms and before they can attain it, while making the end game more fun when you can create fusion reactors but not oxygen on the go.

**Comment by MissCatTrap** [comment]
While not directly related, it is an Evolution to my issue #61 , both making players lives more convenient throughout their game.

**Comment by gisellevonbingen** [comment]
Indeed, I don't have any scenario for how to make player overcome no oxygen yet.
Player must have oxygen for living in my think.

To be that make sense, at least need infinite water source item for crafting that module.
But Ad Astra and this mod don't added it item.

**Comment by gisellevonbingen** [comment]
Enchant/MK2 unit is not planned yet.
Instead, Added creative oxygen can for modpack. fe48e7196639d77a0c33b618519aae0c595ae914
